packages/core/src/config.js

```diff
@@ -25,6 +25,10 @@ export const configSchema = {
       },
       enableJavaScript: {
         type: 'boolean'
+      },
+      scope: {
+        type: 'string',
+        default: ''
```

Comment:
Any reason for this default? Seems like we could remove this and the merge with defaults and it would behave exactly the same.

Reply:
No specific reason -- I thought it needed the default but I now see `enableJavaScript` doesn't have one either